src/mscorlib/src/System/Collections/Concurrent/ConcurrentDictionary.cs

```diff
@@ -1987,14 +1973,7 @@ private ReadOnlyCollection<TValue> GetValues()
         [Conditional("DEBUG")]
         private void Assert(bool condition)
```

Comment:
Should we change the call sites to just use Contract.Assert now that this Assert method is just a wrapper for Contract.Assert?